virttest/libvirt_xml/devices/console.py

```diff
@@ -26,6 +26,8 @@ def __init__(self, type_name='pty', virsh_instance=base.virsh):
                                  has_subclass=True)
         accessors.XMLElementDict('alias', self, parent_xpath='/',
                                  tag_name='alias')
+        accessors.XMLElementDict('log_file', self, parent_xpath='/',
```

Comment:
According to the log definition in libvirt.org, I think it is better to use `log` instead of `log_file`, because it also include other attributes, like `append`. What do you think?

`<log file="/var/log/libvirt/qemu/guestname-serial0.log" append="off"/>`

Reply:
Hi dzheng, Thanks your reminder  ,Done